tools/projmgr/schemas/common.schema.json

```
@@ -445,35 +477,35 @@
     },
     "ProcessorTrustzone": {
       "enum": [ "secure", "secure-only", "non-secure", "off" ],
-      "description": "Select TrustZone mode: secure | secure-only | non-secure | off"
+      "description": "Select TrustZone mode:\n secure | secure-only | non-secure | off ."
     },
     "ProcessorFpu": {
       "enum": [ "dp", "sp", "off" ],
-      "description": "Controls usage of FPU registers (S-Registers that are used for Helium and FPU hardware): dp (double precision) | sp (single precision) | off (disabled)"
+      "description": "Controls usage of FPU instructions:\ndp (double precision)\n sp (single precision)\n off (disabled) ."
     },
     "BuildProcessorFpu": {
       "enum": [ "on", "dp", "sp", "off" ],
-      "description": "Controls usage of FPU registers (S-Registers that are used for Helium and FPU hardware): on (enabled) | dp (double precision) | sp (single precision) | off (disabled)"
+      "description": "Controls usage of FPU instructions:\non (enabled)\n dp (double precision)\n sp (single precision)\n off (disabled) ."
     },
     "ProcessorDsp": {
       "enum": [ "on", "off" ],
-      "description": "Controls usage of DSP: on (enabled) | off (disabled)"
+      "description": "Controls usage of SIMD instructions:\n on (enabled) | off (disabled) ."
```

The `dsp` node concerns DSP extensions, for example disabling the DSP for Cortex-M33 will result in the following AC6 flag:  [`-mcpu=Cortex-M33+nodsp`](https://github.com/Open-CMSIS-Pack/devtools/blob/main/tools/buildmgr/cbuildgen/config/AC6.6.16.2.cmake#L365C25-L365C47)
SIMD flags such as [`-fvectorize`](https://developer.arm.com/documentation/dui0774/l/Compiler-Command-line-Options/-fvectorize---fno-vectorize) are not yet abstracted via dedicated yaml nodes.
```suggestion
      "description": "Controls usage of DSP extensions:\n on (enabled) | off (disabled) ."
```

---

How about: "Controls usage of DSP instruction set extensions by the compiler as supported by the processor"

---

Updated as suggested, but without "by the compiler" as we don't specify that in other features either.